src/components/forms/__tests__/__snapshots__/FormFactory.test.js.snap

```diff
@@ -0,0 +1,186 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I've found that snapshots over 50 lines is usually ignored, if some of the internal snaps are already part of other snapshots maybe they could be avoided?

Reply:
reduced to < 50 lines.`NewVmWizard.test.js.snap` still has 66 lines but I do not see a way to reduce it.

Reply:
> I've found that snapshots over 50 lines is usually ignored, if some of the internal snaps are already part of other snapshots maybe they could be avoided?

This is true for snapshots and other files too. If it looks big and complex, people are less inclined to review and maintain it, until it breaks the build or application itself.

This has already been discussed on `patternfly-react` project, see [my comment on Enzyme & snapshot testing](https://github.com/patternfly/patternfly-react/pull/268#issuecomment-375723540).

In particular:

> as @dmiller9911 pointed out, snapshots shouldn't include fully-resolved HTML tree, otherwise you'll end up with big snapshots and it's hard to differentiate the actual component's HTML bits